guides/micronaut-jms-aws-sqs/java/src/main/java/example/micronaut/SqsClientFactory.java

```diff
@@ -0,0 +1,24 @@
+package example.micronaut;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import io.micronaut.aws.sdk.v1.EnvironmentAWSCredentialsProvider;
+import io.micronaut.context.annotation.Factory;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.env.Environment;
+import jakarta.inject.Singleton;
+
+@Factory  // <1>
+@Requires(notEnv = Environment.TEST)
```

Comment:
This will confuse readers, but actually you don't need it. Simply remove `@Requires` from here. Will add an inline comment in the test factory.